pkg/agent/apiserver/handlers/ovsflows/handler.go

```diff
@@ -70,19 +97,27 @@ func HandleFunc(aq querier.AgentQuerier) http.HandlerFunc {
 		var resps []Response
 		podName := r.URL.Query().Get("pod")
 		namespace := r.URL.Query().Get("namespace")
+		table := r.URL.Query().Get("table")
 
-		if podName == "" && namespace == "" {
-			resps, err = getAllFlows(aq)
+		if podName == "" && namespace == "" && table == "" {
+			resps, err = getFlows(aq, binding.TableIDAll)
 		} else if podName != "" && namespace != "" {
 			// Pod Namespace must be provided to dump flows of a Pod.
 			resps, err = getPodFlows(aq, podName, namespace)
+		} else if table != "" {
+			resps, err = getTableFlows(aq, table)
+			if err == nil && resps == nil {
+				http.Error(w, "invalid table name", http.StatusBadRequest)
+				return
+			}
 		} else {
 			// Not supported.
 			w.WriteHeader(http.StatusBadRequest)
```

Comment:
why don't we use `http.Error` here? (same question for below)

Reply:
We could if we add more information to error messages, which requires to iterate all possible invalidate combinations. Let me know you prefer that way.

Reply:
What's the error message right now on the antctl client side? Do we do any validation there?

Reply:
Error: Bad Request: the server rejected our request for an unknown reason.

Humm... maybe we should add a generic error message here?
BTW, my other PR #645 add errors for one specific invalid input - Namespace not provided, when Pod or NetworkPolicy is specified.

Reply:
Maybe "Unsupported parameter combination, check help message". I am not sure what the best thing to do here is: server-side validation or comprehensive client-side validation. In theory, one could specify any combination of filters (e.g. pod + table) and there is no reason for it to be an error. I guess our choice of *not* using "default" as the default value for namespace makes things a bit harder.

Reply:
"Unsupported parameter combination" sounds good. I feel ideally we should do more validation in both antctl, and at least in server side no need to return detailed error messages.

Reply:
BTW, for "http.StatusNotFound", seems antctl handles it and will print out "resource not found".